src/bin/fxrecorder/config.rs

```diff
@@ -0,0 +1,67 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+use std::net::SocketAddr;
+use std::path::PathBuf;
+
+use serde::Deserialize;
+
+/// The configuration for FxRecorder.
+#[derive(Debug, Deserialize)]
+pub struct Config {
+    /// The address of the `fxrunner` to connect to.
+    pub host: SocketAddr,
+
+    /// The recording configuraton.
+    pub recording: RecordingConfig,
+}
+
+/// Recording-specific configuration.
+#[derive(Clone, Debug, Deserialize)]
+pub struct RecordingConfig {
+    /// The path to the `ffmpeg` executable.`
+    pub ffmpeg_path: PathBuf,
+
+    /// The name of the video capture device.
+    ///
+    /// This can be found via running:
+    /// ```
+    /// ffmpeg -f dshow -list_devices true -i dummy
+    /// ```
+    ///
+    /// This will be used to generate the `-i` argument to `ffmpeg`.
+    pub device: String,
+
+    /// The size of the video stream.
+    ///
+    /// This corresponds to the `-video_size` argument to `ffmpeg`.
+    pub video_size: Size,
+
+    /// The frame rate to capture.
+    ///
+    /// This corresponds to the `-framerate` argument to `ffmpeg`.
+    pub frame_rate: u8,
```

Comment:
While 255fps is probably enough, I wouldn't be opposed to bumping this higher... heh.

Reply:
:)